keyboard/__init__.py

```diff
@@ -251,17 +261,24 @@ def toggle_bt(self):
             self.stop_advertising()
         else:
             self.start_advertising()
+        print(self.current_channel())
+
+    def current_channel(self):
+        if self.ble.connected:
+            return "BT%d" % self.ble_id
+        return "USB"
```

Comment:
USB has higher priority than BT. Should check USB connection first.

Reply:
Updated in new commit.